fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java

```diff
@@ -81,6 +81,14 @@
  */
 public class DataDescription {
     private static final Logger LOG = LogManager.getLogger(DataDescription.class);
+
+    private static final String LINE_DELIMITER = "line_delimiter";
```

Comment:
Could we use the definition in LoadStmt directly?

Reply:
done